tests/test_juv.py

```diff
@@ -851,11 +855,11 @@ def test_stamp_script(
 # ///
 
 
-def main() -> None:                                                                                                                                                                                                               │
-    print("Hello from foo.py!")                                                                                                                                                                                                   │
-                                                                                                                                                                                                                                  │
-                                                                                                                                                                                                                                  │
-if __name__ == "__main__":                                                                                                                                                                                                        │
+def main() -> None:                                                                                                                                                                                                               |
+    print("Hello from foo.py!")                                                                                                                                                                                                   |
+                                                                                                                                                                                                                                  |
+                                                                                                                                                                                                                                  |
+if __name__ == "__main__":                                                                                                                                                                                                        |
```

Comment:
For these snippets, I wonder about setting `encoding='utf-8'` within `Path.open`. E..g.,

```py
        with (tmp_path / "foo.py").open("w", encoding="utf-8") as f:
```

I would prefer to not have the pipes but am also not an encoding expert. The tests still pass for me as is, and I can fiddle around with this myself too so no need to spend a lot of time on it.